docs/more-pushmsg.md

```diff
@@ -33,3 +33,53 @@ func main() {
         url.Values{"message": {"My Message"}, "title": {"My Title"}})
 }
 ```
+
+### PHP (using cURL)
+
+```php
+$data = array(
+    "title"=> "Hello World",
+    "message"=> "Test push From PHP cURL.",
+    "priority"=> 5,
+);
+
+$data_string = json_encode($data);
+
+$url = "http://localhost:8008/message?token=<apptoken>";
+
+$headers = array(
+    'Content-Type: application/json; charset=utf-8'
+);
+
+$ch = curl_init();
+curl_setopt($ch, CURLOPT_URL, $url);
+curl_setopt($ch, CURLOPT_POST, 1);
+curl_setopt( $ch,CURLOPT_HTTPHEADER, $headers );
+curl_setopt( $ch,CURLOPT_RETURNTRANSFER, true );
+curl_setopt( $ch,CURLOPT_POSTFIELDS, $data_string);
+
+$result = curl_exec($ch);
+$code = curl_getinfo($ch, CURLINFO_HTTP_CODE);
+
+curl_close ($ch);
+
+switch ($code) {
```

Comment:
I think we can omit this, the other examples also don't include error handling.

Reply:
I feel it better to understand while using this on PHP Forms